target/linux/ramips/dts/mt7620a_sitecom_wlr-4100v1002.dts

```diff
@@ -0,0 +1,184 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "sitecom,wlr-4100v1002", "ralink,mt7620a-soc";
```

Comment:
Do you really think this is a good spelling?

Reply:
I have no idea. What do you suggest?

Reply:
Well, replacing the spaces with hyphens, like in all of the other examples?
sitecom,wlr-4100-v1-002
But I still doubt we actually need the 002 here ...